src/Compilers/VisualBasic/Portable/Semantics/Conversions.vb

```diff
@@ -1246,6 +1246,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
             ' check arguments against flattened list of target element types 
             Dim result As ConversionKind = wideningConversion
+            Dim involvesNarrowingFromNumericConstant As ConversionKind = Nothing
+            Dim allNarrowingIsFromNumericConstant = ConversionKind.InvolvesNarrowingFromNumericConstant
```

Comment:
There is another ClassifyTupleConversion method that classifies conversions from type. I assume that one does not need aggregation for this bit since this bit can only happen in a conversion from a literal.